src/anomalib/utils/metrics/aupro.py

```diff
@@ -30,6 +32,13 @@ class AUPRO(Metric):
     full_state_update: bool = False
     preds: list[Tensor]
     target: list[Tensor]
+    # When not None, the computation is performed in constant-memory by computing the roc curve
+    # for fixed thresholds buckets/thresholds.
+    # Warning: The thresholds are evenly distributed between the min and max predictions
+    # if all predictions are inside [0, 1]. Otherwise, the thresholds are evenly distributed between 0 and 1.
```

Comment:
isn't it 
```suggestion
    # Warning: The thresholds are evenly distributed between the min and max predictions
    # if all predictions are NOT inside [0, 1]. Otherwise, the thresholds are evenly distributed between 0 and 1.
```
```suggestion
    # Warning: The thresholds are evenly distributed between the min and max predictions
    # if all predictions are NOT inside [0, 1]. Otherwise, the thresholds are evenly distributed between 0 and 1.
```

it's the opposite, no?

Reply:
@jpcbertoldo I guess my proposal is good: 
binary_roc is calling the _binary_precision_recall_curve_format function which is applying the sigmoid on preds when out of [0, 1] before actually computing the roc curve. Thus we want to use thresholds computed from min and max only when the sigmoid is not applied.

Reply:
oh, ok i interpreted the sentence wrong, my bad : )